### PR TITLE
Add 'host' and 'scheme' passing to swagger-jsdoc.

### PR DESCRIPTION
**Pull request by jacekdalkowski**
Makes possibile to pass 'host' and 'scheme' params from 'createSwaggerPage(...)' config options down to swagger-jsdoc, so that target host for 'Try it' button can be different than page host.

### CODE REVIEW

File: typings.d.ts [hunk 1]
```diff
@@ -5,6 +5,8 @@ declare module 'swagger-jsdoc' {
             apis: string[];
             definitions?: {[key: string]: any};
             paths?: {[key: string]: string};
+            host?: string;
+            schemes: string[];
```
Comment (RemyJeancolas) [inline typings.d.ts]: Same here

File: src/index.ts [hunk 2]
```diff
@@ -13,6 +13,8 @@ interface SwaggerPageOptions {
     definitions?: {[key: string]: any};
     routePrefix?: string;
     forceSecure?: boolean;
+    host?: string;
+    schemes: string[];
```
Comment (RemyJeancolas) [inline src/index.ts]: Same remark as before

File: index.d.ts [hunk 3]
```diff
@@ -9,6 +9,8 @@ export interface SwaggerPageOptions {
     definitions?: {[key: string]: any};
     routePrefix?: string;
     forceSecure?: boolean;
+    host?: string;
+    schemes: string[];
```
Comment (RemyJeancolas) [inline index.d.ts]: You should replace the line above with `schemes?: string[];` (note the question mark), this is the reason why the compilation fails during the Travis test.

**Comment by jacekdalkowski** [comment]
Thank you for the response. This is my first GitHub PR and I am sorry for the mess. Thank you for the comments and I am on my way to sorting things out.

**Comment by RemyJeancolas** [comment]
Hi Jacek, no worries, typescript is a bit tricky to understand at the beginning.

By the way, I think I got what you wanted to add in the module, so I took time to add the features you wanted on my own, you should just have to use the last version of the package (`v1.2.0`) to have access to the `description`, `tags`, `host` and `schemes` params.

Let me know if you need something else, I'm trying to keep the repo up to date and aligned with the needs as soon as possible.

**Comment by RemyJeancolas** [comment]
Already implemented since `v1.2.0`

**Pull request closed without merging**